src/AddtodoForm.js

```diff
@@ -23,9 +24,14 @@ function AddTodoForm({onAddTodo}) {
 
     return(
         <form onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle">Title</label>
-            <input id="todoTitle" type="text" name="title" value={todoTitle} onChange={handleTitleChange}></input>
-            <button type="submit">Add</button>
+            <InputWithLabel
+                id="todoTitle"
```

Comment:
I do not think you need id or label. You are not using either on in InputWithLabel

Reply:
Thank you! 😄